src/internal/modules/contact-entry/forms/company-search-select-company.js

```diff
@@ -0,0 +1,50 @@
+const { formFactory, fields } = require('shared/lib/forms');
+const Joi = require('@hapi/joi');
+const { getAddressText } = require('../../address-entry/lib/helpers');
+
+const getCompanyText = row => {
+  return `${row.company.name}, ${row.address ? getAddressText(row.address) : row.company.companyNumber}`;
+};
+
+const getCompanyChoices = companies => {
+  const choices = companies.map(record => {
+    return ({
+      value: record.company.companyNumber,
+      label: getCompanyText(record)
+    });
+  });
+  return choices;
+};
+
+const form = (request, defaultValue) => {
+  const { csrfToken } = request.view;
+  const { sessionKey } = request.query;
+  const { companiesHouseResults } = request.pre;
+
+  const f = formFactory('/contact-entry/new/details/company-search/select-company');
+
+  f.fields.push(fields.radio('selectedCompaniesHouseNumber', {
```

Comment:
According to the prototype this should be a drop down select list and not a radio option

f.fields.push(fields.dropdown

Reply:
Fixed in commit 41818361